autoload/health/check.lua

```diff
@@ -0,0 +1,14 @@
+function nvim_lsp_check_health()
+	local configs = require 'nvim_lsp/configs'
+
+	for _, top_level_config in pairs(configs) do
+		-- the folder needs to exist
+		local new_config = top_level_config.make_config(".")
+		vim.fn['health#report_info']("Checking "..new_config.name)
+
+		local cmd, _ = vim.lsp._cmd_parts(new_config.cmd)
+		if not (vim.fn.executable(cmd) == 1) then
+			vim.fn['health#report_error'](string.format("The given command %q is not executable.", cmd))
```

Comment:
1. brevity
2. the "variable" part of messages should go at the end after a colon, when possible

```suggestion
			vim.fn['health#report_error'](string.format("%s: command is not executable: %q", new_config.name, cmd))
```